US-CBP/cbp-theme
# feat: automated npm publishing

- cleanup of author names
- added README
- minor .gitignore additions dist, cypress, etc
- added dist prepare task to copy readme, proper version, and package.json to dist in root
- refactored npm css, website tasks names
- add arguments to automate publishing to npm on a tag via travis and encrypted npm token with beta tag

#### What's this PR do?

This helps automated the process of npm publishing via Travis CI

#### Where should the reviewer start?

take a look at the package.json, .travis.yml, publish.sh and 

https://docs.travis-ci.com/user/deployment/npm/

#### How should this be manually tested?

once merged we will test using npm version which will create a tag and should kick off a build plus publish to npm

#### Any background context you want to provide? N/A
#### What are the relevant tickets? N/A
#### Screenshots (if appropriate) 
#### Questions:
- Is there a blog post? N/A
- Does the knowledge base need an update? We will need to update the README for developers wishing to understand publishing the design system
